libs/agno/agno/os/routers/session/session.py

```diff
@@ -100,18 +104,25 @@ async def get_sessions(
             alias="type",
             description="Type of sessions to retrieve (agent, team, or workflow)",
         ),
+        minimal: bool = Query(
+            default=False,
+            description="If true, return minimal session info (session_id, session_name, session_state, timestamps). "
+            "If false, return full session details including chat_history, metrics, and metadata.",
+        ),
```

Comment:
Nice, I like it. 

Although probably the default should be to return the minimal/summary response in the list endpoints? then we have `full` as opt-in for the extended version?